app/views/docs/configuration.phtml

```diff
@@ -22,10 +22,143 @@
 <h2><a href="#git" id="git">Configure GitHub App</a></h2>
 <p>
     Appwrite supports automatic deployments through Git integration. 
-    In order for Appwrite to access your repos, you must configure a GitHub app to enable this integration.
+    In order for Appwrite to access your repos, you must <a href="https://docs.github.com/en/apps/creating-github-apps/about-creating-github-apps/about-creating-github-apps" target="_blank" rel="noopener">create a GitHub app</a> to enable this integration.
+    The GitHub app requires the following configurations.
 </p>
 
-<p>You'll have to configure the following environment variables.</p>
+<p>
+    For automatic Git deployment to work, Appwrite needs to receive communication from GitHub, this means your Appwrite project must be accessible on the internet.
+    If you're running on <code>localhost</code>, you need to run a proxy like <b>ngrok</b>.
+</p>
+
+<h3><a href="#callback" id="callback">Callback URL</a></h3>
+<p>
+    GitHub will use <b>callback URLs</b> to redirect users back to Appwrite. 
+    Set these callback URLs under <b>Identifying and authorizing users</b>.
+</p>
+<table class="full">
+    <thead>
+        <tr>
+            <td>URLs</td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>http://[HOSTNAME_OR_IP]/v1/vcs/github/callback</code></td>
+        </tr>
+        <tr>
+            <td><code>http://[HOSTNAME_OR_IP]/v1/account/sessions/oauth2/callback/github/console</code></td>
+        </tr>
+    </tbody>
+</table>
+
+<h3><a href="#post-install" id="post-install">Post Installation</a></h3>
+<p>
+    Check the <b>Redirect on update</b> box under the <b>Post installation</b> section.
```

Comment:
This is lacking context as to which page the developer needs to check this option in. We should attach a screenshot or provide a link to the page in the previous section. 

Something like "Head to https://github.com/path/to/create/app and do ... "

Reply:
Oh your right, the settings link is static good call

Reply:
Alright then 👍